web/themes/contrib/civictheme/templates/webform/webform.html.twig

```diff
@@ -0,0 +1,19 @@
+{#
```

Comment:
@govindmaloo 
I'm not sure what is going on here.

We already have Webform support when it is used within a paragraph.

`paragraph--civictheme-webform.html.twig`

We do not support standalone webforms, because it is hard to control.

Can you please explain why this is needed

Reply:
In both cases, we would need a form with the required classes. Without the classes, state js will not work. In order to render form with the required classes we need this template.